packages/machine/src/protocol/install-virtual-app.ts

```diff
@@ -73,7 +73,8 @@ export const INSTALL_VIRTUAL_APP_PROTOCOL: ProtocolExecutionFlow = {
       context.outbox[0].toAddress = params2.respondingAddress;
       context.outbox[0].signature = context.signature3; // s6
     },
-    Opcode.IO_SEND,
+
+    Opcode.IO_SEND_AND_WAIT,
```

Comment:
I'm pretty sure this should just be a `SEND`. No one else will send a message to the intermediary.

Reply:
Ah, okay